filebeat/_meta/fields.common.yml

```diff
@@ -100,3 +100,83 @@
       description: >
         Content length of the HTTP response body.
 
+    - name: source_ecs
```

Comment:
I'm not sure if I'm missing anything, but your `source` and `destination` objects are only comprised of `ip` and `geo` at the moment, correct? But why not add [the other ones](https://github.com/elastic/ecs#destination)? `port`, `mac`, `hostname`, `domain` and `subdomain` come to mind.

Suricata doesn't give you the mac address, even the local one, so you won't populate it. That's fine. But `filebeat/_meta/fields.common.yml` is not Suricata specific, it should be all the common fields.

Or are you intentionally adding only the ones you need for Suricata, for the moment? If so, you'll still need `port`. Ideally you'd add one for the domain as well, but right now we're still debating the details of what things should be named there (discussion in https://github.com/elastic/ecs/issues/84)

Reply:
ECS fields are already added to every Beat (introduced in #8313)

So what I'm doing here with `destination` is just to inject a `geo` field, which is missing from ECS. The final template will have all fields + geo. In `source_ecs` I just added the fields I needed. I will re-check to see which ones I'm missing apart from port. Thanks.

Reply:
> inject a geo field, which is missing from ECS

Yes, this is on the shortlist of the next things we're doing. It's one of the "reuseable" objects, and we need to come up with a new way to specify them, so they're added to all of the expected places in the index template :-)

Reply:
I would say to not block this to add them first in Beats directly until ECS figured it out